### DISCUSSION TITLE
Short video of training.

### DISCUSSION THREAD

**Discussion by jjlynn27** [trackmania-rl/tmrl#83] [Show and tell]
https://youtu.be/J1oT3ukHLQo
Link to something I made today. Call me Mr Scorsese. Or Ray Charles.

**Comment by koi0823** [comment]
hey bro can we get a connection for this i am degree student in computer science and major on AI i want to ask how u set up the enviroment and gym and parameter so can give me a guide for project as a final year project 

and also how u make a multiple training sessions simultaneously to train
![Screenshot 2024-04-29 010302](https://github.com/trackmania-rl/tmrl/assets/86875051/340ed5dc-748e-4726-ac97-df53432ad75c)

if u dont might can add my discord 
koihaha#5605
